Atomic/mount.py

```diff
@@ -140,14 +140,33 @@ def set_args(self, args):
     def _info(self):
         return self.d.info()
 
-    def mount(self):
+    def _try_ostree_mount(self, best_mountpoint_for_storage):
+        if best_mountpoint_for_storage:
+            mountpoint = os.path.join(self.syscontainers.get_ostree_repo_location(), "tmp", str(os.getpid()), self.image)
```

Comment:
Using `getpid()` in here is going to make it hard to avoid leaking them.  I'd make it `tmp/atomic-mount` or something.  Then it's easy to have a cleanup function delete that directory.  This is what we do in rpm-ostree.  Now, *inside* that directory, maybe you do want a pid.